src/APIRequests.js

```diff
@@ -1,24 +1,11 @@
-import React, { Component } from 'react'
-
-class APIRequests extends Component {
-    componentDidMount() {
-        const apiURL = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
-        fetch(apiURL) 
-            .then((response) => response.json())
-            .then((response) => console.log(response))
-            .catch(err => console.log(err))
-    }
-
-    render() {
-        return 'hi'
-    }
+export const getAllMovies = async () => {
+    const response = await fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    const data = await response.json()
+    return data
 }
 
-export default APIRequests
-
-// const getAllMovies = () => {
-//     return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-//     .then((response) => response.json())
-//     .then((response) => console.log(response))
-//     .catch(err => console.log(err))
-// }
\ No newline at end of file
+export const getSingleMovieDetails = async (movieID) => {
+    const response = await fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${movieID}`)
+    const data = await response.json();
+    return data;
+  }
```

Comment:
Nice work getting these methods hooked up! I wonder if there is a way that we can combine these methods & pass in everything after `/v2/` as an argument...